service/controller/resource/s3object/desired.go

```diff
@@ -31,12 +31,10 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
-	releaseVersion := key.ReleaseVersion(cr)
-
 	var cluster infrastructurev1alpha2.AWSCluster
 	var clusterCerts certs.Cluster
 	var clusterKeys randomkeys.Cluster
-	var release *v1alpha1.Release
+	var release *releasev1alpha1.Release
```

Comment:
Can you revert all changes in this file? They do not seem necessary and I see conflicts down the road with my own ugly PR.

Reply:
Done.